packages/file_selector/file_selector_windows/windows/test/test_file_dialog_controller.h

```diff
@@ -51,14 +51,18 @@ class TestFileDialogController : public FileDialogController {
   ~TestFileDialogController();
 
   // FileDialogController:
+  HRESULT SetFolder(IShellItem* folder) override;
   HRESULT SetFileTypes(UINT count, COMDLG_FILTERSPEC* filters) override;
   HRESULT SetOkButtonLabel(const wchar_t* text) override;
   HRESULT Show(HWND parent) override;
   HRESULT GetResult(IShellItem** out_item) const override;
   HRESULT GetResults(IShellItemArray** out_items) const override;
 
   // Accessors for validating IFileDialogController setter calls.
-  std::wstring GetDefaultFolderPath() const;
+  // Get the folder path set by FileDialogController::SetFolder
```

Comment:
I think it's worth explaining here also where there are two getters:

```
// Gets the folder path set by SetFolder.
//
// This exists because there are multiple ways that the value returned by
// GetDialogFolderPath can be changed, so this allows specifically validating
// calls to SetFolder.
```